### PR TITLE
1.10: settings.py refactor

### PR DESCRIPTION
**Pull request by valentijnscholten**

### CODE REVIEW

File: docs/api-docs.rst [hunk 1]
```diff
@@ -2,8 +2,8 @@ DefectDojo API Documentation
 ============================
 
 .. warning::
-   API v1 is deprecated and EOS is on 12-31-2020.
-   EOL is on 06-30-2021.
+   API v1 is deprecated and no longer maintained aftger on 12-31-2020.
```
Comment (Maffooch) [inline docs/api-docs.rst]: Got a typo here

File: docs/getting-started.rst [hunk 2]
```diff
@@ -5,20 +5,24 @@ Docker Compose Install (recommended)
...
 * Select the appropriate branch you're working on
-* Instructions in the `KUBERNETES.md` file at the root of the repository.
+* Instructions in the [`KUBERNETES.md`](https://github.com/DefectDojo/django-DefectDojo/blob/master/KUBERNETES.md) file at the root of the repository.
 
-Setup.bash Install 
-******************
+Setup.bash Install (no longer maintained)
+*****************************************
 .. warning::
    This installation method will be EOL on 2020-12-31
```
Comment (Maffooch) [inline docs/getting-started.rst]: Should probably clarify EOL means its removed from the repo like it was for APIv1

**Comment by valentijnscholten** [comment]
thanks for reviewing, did some updates.

**Review by Maffooch — APPROVED** [review]

File: docs/api-docs.rst [hunk 3]
```diff
@@ -2,8 +2,8 @@ DefectDojo API Documentation
 ============================
 
 .. warning::
-   API v1 is deprecated and EOS is on 12-31-2020.
-   EOL is on 06-30-2021.
+   API v1 is deprecated and will be no longer maintained after 12-31-2020.
```
Comment (madchap) [inline docs/api-docs.rst:5]: will no longer be maintained.

File: docs/getting-started.rst [hunk 4]
```diff
@@ -5,20 +5,24 @@ Docker Compose Install (recommended)
...
-* Instructions in the `KUBERNETES.md` file at the root of the repository.
+* Instructions in the [`KUBERNETES.md`](https://github.com/DefectDojo/django-DefectDojo/blob/master/KUBERNETES.md) file at the root of the repository.
 
-Setup.bash Install 
-******************
+Setup.bash Install (no longer maintained)
+*****************************************
 .. warning::
-   This installation method will be EOL on 2020-12-31
+   This installation method will is EOL and will be removed on 2020-12-31
```
Comment (madchap) [inline docs/getting-started.rst:19]: remove extra "will"

**Pull request merged**